src/app/util/af-event.cpp

```diff
@@ -67,57 +89,28 @@ uint16_t emAfAppEventContextLength        = EMBER_AF_EVENT_CONTEXT_LENGTH;
 EmberAfEventContext emAfAppEventContext[] = { EMBER_AF_GENERATED_EVENT_CONTEXT };
 #endif // EMBER_AF_GENERATED_EVENT_CONTEXT
 
-const char * emAfEventStrings[] = { EM_AF_SERVICE_DISCOVERY_EVENT_STRINGS
+const char * emAfEventStrings[] = {
 
 #ifdef EMBER_AF_GENERATED_EVENTS
-                                        EMBER_AF_GENERATED_EVENT_STRINGS
-#endif
-
-#ifdef EMBER_AF_PLUGIN_FRAGMENTATION
-                                            EMBER_AF_FRAGMENTATION_EVENT_STRINGS
+    EMBER_AF_GENERATED_EVENT_STRINGS
 #endif
 
-                                                EMBER_AF_TEST_HARNESS_EVENT_STRINGS };
+};
 
-EmberEventData emAfEvents[] = { EM_AF_SERVICE_DISCOVERY_EVENTS
+EmberEventData emAfEvents[] = {
 
 #ifdef EMBER_AF_GENERATED_EVENTS
-                                    EMBER_AF_GENERATED_EVENTS
+    EMBER_AF_GENERATED_EVENTS
 #endif
 
-#ifdef EMBER_AF_PLUGIN_FRAGMENTATION
-                                        EMBER_AF_FRAGMENTATION_EVENTS
-#endif
-
-                                            EMBER_KEY_ESTABLISHMENT_TEST_HARNESS_EVENT
-
-                                { NULL, NULL } };
+    { NULL, NULL }
+};
 
 const char emAfStackEventString[] = "Stack";
 
 // *****************************************************************************
 // Functions
 
-// A function used to initialize events for idling
-void emAfInitEvents(void)
```

Comment:
This is called from `emberAfInit`.  We should leave it here but make it a no-op, or remove that (commented-out) call.

Reply:
choosing no-op